apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/direct/FissClaimRdaSink.java

```diff
@@ -41,15 +42,18 @@ public RdaChange<RdaFissClaim> transformMessage(String apiVersion, FissClaimChan
   }
 
   @Override
-  RdaApiClaimMessageMetaData createMetaData(RdaChange<RdaFissClaim> change) {
+  RdaClaimMessageMetaData createMetaData(RdaChange<RdaFissClaim> change) {
     final RdaFissClaim claim = change.getClaim();
-    return RdaApiClaimMessageMetaData.builder()
+    return RdaClaimMessageMetaData.builder()
         .sequenceNumber(change.getSequenceNumber())
         .claimType(RdaApiProgress.ClaimType.FISS)
         .claimId(claim.getDcn())
         .mbiRecord(claim.getMbiRecord())
         .claimState(String.valueOf(claim.getCurrStatus()))
         .receivedDate(claim.getLastUpdated())
+        .locations(
+            new StringList().addIfNonNull(claim.getCurrLoc1()).addIfNonEmpty(claim.getCurrLoc2()))
```

Comment:
why the difference between the checks (ifNonNull vs ifNonEmpty) between loc1 and loc2?  Aren't both columns NOT NULL?

Reply:
`currLoc2` is not nullable but it could theoretically contain an empty string.  The transformer should not allow empty values to be transformed.  If the transformer ever did fail to stop an empty string would we want to include the empty string in the locations array?

Reply:
For this table, no.  I don't believe the empty string or NULL value provides any value for our analysts